src/components/Community/Events/index.tsx

```diff
@@ -8,6 +8,7 @@ import Link from '../../Link'
 import Block from '../Block'
 import Section from '../Section'
 import { logEvent } from '../../../utils/front/ga'
+import { isExpired } from '../../../utils/shared/expiration.js'
```

Comment:
how much does it affect the bundle size ? (just thinking if it makes sense to filter them built time)

Reply:
I would *love* to do this at build time! There's certainly a cost to doing this in the component, it's just the quickest route. `isExpired` works at build time so I can pivot to using this there. It'll take some more code but be worth it in the long run IMO.